mocca-client/src/main/java/com/paypal/mocca/client/MoccaSerializer.java

```diff
@@ -355,7 +355,9 @@ private String objectToString(final Object object, final String name, final List
         } else if (object instanceof Optional) {
             Optional<?> optional = (Optional) object;
             return optional.map(v -> objectToString(v, name, ignoreFields)).orElse("null");
-        } else {
+        } else if (object instanceof Enum) {
```

Comment:
Undo this change please, and add an enum condition in line 345

Reply:
Done